docs/upgrade/0.4.x_to_0.5.rst

```diff
@@ -0,0 +1,76 @@
+Upgrade from 0.4.x to 0.5.x
+===========================
+
+Beginning with SecureDrop 0.5, the source and journalist interfaces
+are localized. After an unattended upgrade of the application server,
+these translations are available but are not activated by default.
+
+.. note::
+   See the :doc:`installation documenation <../install>` for a list
+   of supported languages.
+
+The steps below should be performed on **the Admin Workstation**
+associated with your SecureDrop instance.
+
+Pull the latest release
+-----------------------
+
+Open a **Terminal** and navigate to your SecureDrop directory.
+
+.. code:: sh
+
+   cd ~/Persistent/securedrop
+
+Stash your local configuration, fetch the latest code, and verify the tag for the
+latest release (0.5):
+
+.. code:: sh
+
+   git stash save "site specific configs"
+   git fetch
+   git tag -v 0.5
+
+The output of the above commands should include ``Good signature from
+"SecureDrop Release Signing Key"``. If it does
+not, please contact us immediately at support@freedom.press.
+
+.. note::
+  You may also see output from GPG warning you that the key is not certified
+  with a trusted signature. This means that there is not a trust path to the
+  release signing key. As long as you see the fingerprint ``2224 5C81 E3BA EB41
+  38B3 6061 310F 5612 00F4 AD77`` displayed and the signature verifies as
+  described above then you can proceed safely.
+
+Once you've verified the latest release, check it out, then pop your local
+configuration back into place:
+
+.. code:: sh
+
+   git checkout 0.5
+   git stash pop
+
+Choose the list of supported languages
+--------------------------------------
+
+You need to run the `./securedrop-admin sdconfig` command again,
+following the same instructions as during the :doc:`fisrt installation
```

Comment:
typo here

Reply:
I don't  see it ? But I'm not a native english speaker, that may be why ;-)

Reply:
damn s/fisrt/first/ :-)